geoapi/tests/external_data_tests/test_external_data.py

```diff
@@ -482,17 +483,35 @@ def test_get_additional_files_with_available_files(agave_utils_with_geojson_file
     available_files = ["/testPath/file.shx",
                        "/testPath/file.dbf",
                        "/testPath/file.prj"]
-    files = get_additional_files("testSystem",
+    files = get_additional_files(shapefile_fixture,
+                                 "testSystem",
                                  "/testPath/file.shp",
                                  agave_utils_with_geojson_file,
                                  available_files=available_files)
     assert len(files) == 3
 
 
-def test_get_additional_files_but_missing_prj(agave_utils_with_geojson_file):
+def test_get_additional_files_shapefiles_but_missing_prj(shapefile_fixture, agave_utils_with_geojson_file):
     available_files_missing_prj = ["/testPath/file.shx", "/testPath/file.dbf"]
     with pytest.raises(Exception):
-        get_additional_files("testSystem",
+        get_additional_files(shapefile_fixture,
+                             "testSystem",
                              "/testPath/file.shp",
                              agave_utils_with_geojson_file,
                              available_files=available_files_missing_prj)
+
+
+def test_get_additional_files_rapid_questionnaire(questionnaire_file_with_assets_fixture, agave_utils_with_geojson_file):
```

Comment:
Should this be `test_get_additional_files_rapid_questionnaire_with_assets` to be explicit?

Reply:
done 👍